README.md

```diff
@@ -48,7 +48,7 @@ When publishing a shared library to npm, be sure to include `mozillafoundation`
 
 ## GitHub Flow
 
-Active projects and future projects will use GitHub flow for branching. This means that new features should be written in feature branches based on `master`. When ready, open a pull request against `master` to have the new feature (or fix) brought into the base branch. Branches should only merge into master when fully completed, so other people's work is never blocked.
+Active projects and future projects will use GitHub flow for branching. This means that new features should be written in feature branches in the same repo, branched from `master`. When ready, open a pull request against `master` to have the new feature (or fix) brought into the base branch. Branches should only merge into master when fully completed, so other people's work is never blocked.
```

Comment:
I tried to cover that later on, but good point. "in the same repo (if you have permission), branched from `master`"? Might still be implied to be limiting...